plugin/src/main/java/net/aufdemrand/denizen/objects/properties/trade/TradeResult.java

```diff
@@ -0,0 +1,83 @@
+package net.aufdemrand.denizen.objects.properties.trade;
+
+import net.aufdemrand.denizen.objects.dItem;
+import net.aufdemrand.denizen.objects.dTrade;
+import net.aufdemrand.denizencore.objects.Mechanism;
+import net.aufdemrand.denizencore.objects.dObject;
+import net.aufdemrand.denizencore.objects.properties.Property;
+import net.aufdemrand.denizencore.tags.Attribute;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.MerchantRecipe;
+
+public class TradeResult implements Property {
+
+    public static boolean describes(dObject recipe) {
+        return recipe instanceof dTrade;
+    }
+
+    public static TradeResult getFrom(dObject recipe) {
+        if (!describes(recipe)) {
+            return null;
+        }
+        return new TradeResult((dTrade) recipe);
+    }
+
+    private dTrade recipe;
+
+    public TradeResult(dTrade recipe) {
+        this.recipe = recipe;
+    }
+
+    public String getPropertyString() {
+        if (recipe.getRecipe() == null) {
+            return null;
+        }
+        return (new dItem(recipe.getRecipe().getResult())).identify();
+    }
+
+    public String getPropertyId() {
+        return "result";
+    }
+
+    public String getAttribute(Attribute attribute) {
+        if (attribute == null) {
+            return null;
+        }
+
+        // <--[tag]
+        // @attribute <trade@trade.result>
+        // @returns dItem
+        // @mechanism dTradeRecipe.result

```

Comment:
Bad meta

Reply:
Oops, forgot to change that from the old (bad) PR. Will fix.